azurerm_application_insights_web_test/main.tf

```diff
@@ -37,7 +37,8 @@ resource "azurerm_application_insights_web_test" "web_test" {
 	   Description="" CredentialUserName="" CredentialPassword="" PreAuthenticate="True" Proxy="default"
 	   StopOnError="False" RecordedResultFile="" ResultsLocale="">
     <Items>
```

Comment:
what is this GUID? Why is it hardcoded?

Reply:
We already went through that in previous conversations. It's not something IO related. It's part of the .net web-test.
https://docs.microsoft.com/en-us/dotnet/api/microsoft.visualstudio.testtools.webtesting.webtest.guid?view=visualstudiosdk-2017